flexmeasures/ui/templates/base.html

```diff
@@ -806,6 +806,22 @@
     </script>
 
     <script>
+        // Format ISO 8601 duration using luxon
+        function formatResolution(isoDuration) {
```

Comment:
I believe my original comment was to move it to a js util function module. A template is not the most natural place for util functions.